src/phoenix/server/api/mutations/api_key_mutations.py

```diff
@@ -151,10 +151,21 @@ async def create_user_api_key(
     @strawberry.mutation(permission_classes=[HasSecret, IsAuthenticated])  # type: ignore
     async def delete_system_api_key(
         self, info: Info[Context, None], input: DeleteApiKeyInput
-    ) -> DeleteSystemApiKeyMutationPayload:
+    ) -> DeleteApiKeyMutationPayload:
         assert (token_store := info.context.token_store) is not None
         api_key_id = from_global_id_with_expected_type(
             input.id, expected_type_name=SystemApiKey.__name__
         )
         await token_store.revoke(ApiKeyId(api_key_id))
-        return DeleteSystemApiKeyMutationPayload(id=input.id, query=Query())
+        return DeleteApiKeyMutationPayload(id=input.id, query=Query())
+
+    @strawberry.mutation(permission_classes=[HasSecret, IsAuthenticated])  # type: ignore
```

Comment:
```suggestion
    @strawberry.mutation(permission_classes=[HasSecret, IsAuthenticated, IsAdmin])  # type: ignore
```
Need to be an admin to delete a user's key

Reply:
oh nice this wasn't in main and misse it when i switched over